src/bootstrap/check.rs

```diff
@@ -110,6 +110,9 @@ pub fn compiletest(build: &Build,
 
     if let Some(nodejs) = build.config.nodejs.as_ref() {
         cmd.arg("--nodejs").arg(nodejs);
+    } else {
+        // FIXME: Print a message about being unable to detect a nodejs binary;
+        // and suggest a fix (passing --nodejs to configure?).
```

Comment:
I think it's fine to leave this kind of error reporting to compiletest rather than in rustbuild itself